### PR TITLE
Helm installation method for Palette

### PR DESCRIPTION
**Pull request by stenwt**
## Describe the Change

This PR adds details on the Helm installation method of Palette. User can use this guide to install a self-hosted instance of Palette onto a Kubernetes cluster.

## Review Changes

💻 [Preview](https://deploy-preview-1220--docs-spectrocloud.netlify.app/enterprise-version/deploying-palette-with-helm)

🎫 [Jira Ticket](https://spectrocloud.atlassian.net/browse/DOC-746)

### CODE REVIEW

**Comment by spectro-prow** [comment]
Welcome @stenwt! It looks like this is your first PR to spectrocloud/librarium 🎉

**Comment by spectro-prow** [comment]
Hi @stenwt. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: content/docs/12-enterprise-version.md
Comment (karl-cardenas-coding) [inline content/docs/12-enterprise-version.md:17]:
```suggestion
## VMware Quick Start Mode
```

File: content/docs/12-enterprise-version.md
Comment (karl-cardenas-coding) [inline content/docs/12-enterprise-version.md:21]:
```suggestion
## VMware Enterprise Mode
```

File: content/docs/12-enterprise-version.md
Comment (karl-cardenas-coding) [inline content/docs/12-enterprise-version.md:25]:
```suggestion
## Helm Chart
```

File: content/docs/12-enterprise-version/03-deploying-palette-with-helm.md
Comment (karl-cardenas-coding) [inline content/docs/12-enterprise-version/03-deploying-palette-with-helm.md:2]:
```suggestion
title: "Helm Chart Mode"
```

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]

File: content/docs/12-enterprise-version/03-deploying-palette-with-helm.md
Comment (stenwt) [reply content/docs/12-enterprise-version/03-deploying-palette-with-helm.md:26]: yes

File: content/docs/12-enterprise-version/03-deploying-palette-with-helm.md
Comment (karl-cardenas-coding) [inline content/docs/12-enterprise-version/03-deploying-palette-with-helm.md:44]:
```suggestion
Palette cannot manage the cluster its installed onto due to component conflicts. Consider using a managed Kubernetes service to minimize management overhead. The Palette Helm Chart is not tied to any particular managed Kubernetes service.
```

File: content/docs/12-enterprise-version.md
Comment (ritawatson) [inline content/docs/12-enterprise-version.md:23]:
```suggestion
A highly available multi-node Palette installation that is typically used for production purposes. Check out the [Enterprise Mode](/enterprise-version/deploying-an-enterprise-cluster) guide to get started.
```

File: content/docs/12-enterprise-version/00-on-prem-system-requirements.md
Comment (ritawatson) [reply content/docs/12-enterprise-version/00-on-prem-system-requirements.md:19]: Our internal style uses "on-prem" instead of the spell out.

File: content/docs/12-enterprise-version/00-on-prem-system-requirements.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/00-on-prem-system-requirements.md:25]:
```suggestion
| **Helm Chart Mode**  | Install Palette in an existing Kubernetes cluster using a Helm Chart. |
```

File: content/docs/12-enterprise-version/03-deploying-palette-with-helm.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/03-deploying-palette-with-helm.md:4]:
```suggestion
metaDescription: "Learn how to deploy Palette SaaS in a Kubernetes cluster using a Helm Chart."
```

File: content/docs/12-enterprise-version/03-deploying-palette-with-helm.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/03-deploying-palette-with-helm.md:2]:
```suggestion
title: "Install Palette with Helm Chart"
```

File: content/docs/12-enterprise-version/03-deploying-palette-with-helm.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/03-deploying-palette-with-helm.md:18]:
```suggestion
You can use the Palette Helm Chart to install Palette in a multi-node Kubernetes cluster in your production environment.
```

File: content/docs/12-enterprise-version/03-deploying-palette-with-helm.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/03-deploying-palette-with-helm.md:20]:
```suggestion
This installation method is common in secure environments with restricted network access that prohibits using Palette SaaS. Review our [architecture diagrams](/architecture/networking-ports) to ensure your Kubernetes cluster has the necessary network connectivity for Palette to operate successfully. 
```

File: content/docs/12-enterprise-version/03-deploying-palette-with-helm.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/03-deploying-palette-with-helm.md:49]:
```suggestion
Palette cannot manage the cluster that it is installed onto due to component conflicts. Consider using a managed Kubernetes service to minimize management overhead. The Palette Helm Chart is not tied to any particular managed Kubernetes service.
```

File: content/docs/12-enterprise-version/03-deploying-palette-with-helm.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/03-deploying-palette-with-helm.md:57]:
```suggestion
Choose the installation steps for your target environment. The steps in the generic tab apply to all Kubernetes clusters. Steps in other tabs have instructions explicitly tailored to the target environment.
```

File: content/docs/12-enterprise-version/03-deploying-palette-with-helm.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/03-deploying-palette-with-helm.md:83]:
```suggestion
3. Review the **values.yaml** . You must populate the `env.rootDomain` parameter to the domain you will use for the installation. All other parameter values are optional, and you can reset or change them with a Helm upgrade operation.
```

File: content/docs/12-enterprise-version/03-deploying-palette-with-helm.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/03-deploying-palette-with-helm.md:154]:
```suggestion
    Change `--region` and `--nodes` as required. You can also change the instance size. 
    
    Note that the [minimum instance requirement](https://aws.amazon.com/ec2/instance-types/) is three nodes with a least  4 CPUs and 12 GB of Memory per node. 
```

File: content/docs/12-enterprise-version/03-deploying-palette-with-helm.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/03-deploying-palette-with-helm.md:157]:
```suggestion
3. When the cluster is available, go ahead and configure the OpenID Connect (OIDC) for the cluster to use Palette as the Identity Provider (IDP).
```

File: content/docs/12-enterprise-version/03-deploying-palette-with-helm.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/03-deploying-palette-with-helm.md:180]:
```suggestion
6. Select the **palette-selfhost** cluster to access its details page. 
```

File: content/docs/12-enterprise-version/03-deploying-palette-with-helm.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/03-deploying-palette-with-helm.md:192]:
```suggestion
9. Search for the **AmazonEBSCSIDriverPolicy** policy and add it to the role. 
```

File: content/docs/12-enterprise-version/03-deploying-palette-with-helm.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/03-deploying-palette-with-helm.md:202]:
```suggestion
10. Extract the Helm Chart files from the compressed asset we provided to you. Replace the file path and version place holder as needed.
```

File: content/docs/12-enterprise-version/03-deploying-palette-with-helm.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/03-deploying-palette-with-helm.md:218]:
```suggestion
12. Review the **values.yaml** . You must populate the `env.rootDomain` parameter to the domain you will use for the installation. In addition, add the same `rootDomain` with port `:4222` to the `natsUrl` in the `nats` section of the YAML. Example: `env.rootDomain: my-domain.com:4222`.
```

File: content/docs/12-enterprise-version/03-deploying-palette-with-helm.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/03-deploying-palette-with-helm.md:220]:
```suggestion
    All other parameter values are optional, and you can reset or change them with a Helm upgrade operation.
```

File: content/docs/12-enterprise-version/03-deploying-palette-with-helm.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/03-deploying-palette-with-helm.md:224]:
```suggestion
 13. If you wish to use [AWS ACM for SSL Certs](https://docs.aws.amazon.com/acm/latest/userguide/acm-overview.html), instead of the default self-signed certificate that the Nginx *ingress controller* generates, you can add it to the `annotations` under `ingress`.
```

File: content/docs/12-enterprise-version/03-deploying-palette-with-helm.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/03-deploying-palette-with-helm.md:270]: Should the doc refer to ingress-nginx-controller as a load balancer? Isn't it ingress-nginx-controller OR load balancer? (could be my misunderstanding)

File: content/docs/12-enterprise-version/03-deploying-palette-with-helm.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/03-deploying-palette-with-helm.md:302]:
```suggestion
`env.rootDomain` parameter in the **values.yaml**.
```

File: content/docs/12-enterprise-version/03-deploying-palette-with-helm.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/03-deploying-palette-with-helm.md:301]:
```suggestion
You can validate that the installation of Palette is successful by visiting the custom domain you assigned to the 
```

File: content/docs/12-enterprise-version/03-deploying-palette-with-helm.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/03-deploying-palette-with-helm.md:321]:
```suggestion
Start exploring the Palette system dashboard so that you become familiar with the available actions you can take as an administrator. Check out the [System Console Dashboard](/enterprise-version/system-console-dashboard) resource to learn more.
```

**Review by ritawatson — APPROVED** [review]
A few suggestions.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1220#pullrequestreview-1369502766" title="Approved">ritawatson</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1220#" title="Author self-approved">stenwt</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/12-enterprise-version/00-on-prem-system-requirements.md
Comment (karl-cardenas-coding) [reply content/docs/12-enterprise-version/00-on-prem-system-requirements.md:19]: Nice catch Rita

File: content/docs/12-enterprise-version/03-deploying-palette-with-helm.md
Comment (karl-cardenas-coding) [inline content/docs/12-enterprise-version/03-deploying-palette-with-helm.md:176]:
```suggestion
5. Log in to the [AWS console](https://console.aws.amazon.com) and navigate to the EKS Dashboard.
```

File: content/docs/12-enterprise-version/03-deploying-palette-with-helm.md
Comment (karl-cardenas-coding) [inline content/docs/12-enterprise-version/03-deploying-palette-with-helm.md:250]:
```suggestion
        # Used to terminate HTTPS traffic at the load balancer versus passing through the load balancer. This parameter is available in Palette 3.3 or greater.
```

**Review by karl-cardenas-coding — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1220#pullrequestreview-1369774988" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1220#pullrequestreview-1369502766" title="Approved">ritawatson</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1220#" title="Author self-approved">stenwt</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
@stenwt: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | 59c9a9a34309dfdd5eec44dcfebcb71b0d1af3ca | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1220/librarium_build/1643336303959347200) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Pull request merged**
